dnplab/io/bes3t.py

```diff
@@ -63,12 +63,13 @@ def import_bes3t(path):
         raise TypeError("data file must be .DTA, .DSC, .YGF, or .ZGF")
 
     attrs = load_dsc(path_dsc)
+
     values, dims, coords, attrs = load_dta(
         path_dta, path_xgf, path_ygf, path_zgf, attrs
     )
-
-    # Assign data/spectrum type
+    attrs["spectrometer_format"] = "xepr"
     attrs["experiment_type"] = "epr_spectrum"
```

Comment:
You can't assume that bes3t will be an "epr_spectrum" experiment type. Many other types of EPR data can also be stored in bes3t format.

Reply:
@ehuang94